integrations/gcp/gcp-storage/README.md

```diff
@@ -0,0 +1,99 @@
+## Overview
+
+<!-- Sensu Integration description; supports markdown -->
+
+The [GCP Storage] monitoring integration collects GCP Storage metrics from [GCP Monitoring API], and alerts on various conditions.
+
+[GCP Storage]: https://cloud.google.com/storage/
+[GCP Monitoring API]: https://cloud.google.com/monitoring/api/v3
+
+<!-- Provide a high level overview of the integration contents (e.g. checks, filters, mutators, handlers, assets, etc) -->
+
+This integration includes the following resources:
+
+* `gcp-storage-metrics` [check]
+* `sensu/gcp-monitoring-check:0.2.0` [asset]
+
+## Dashboards
+
+<!-- List of compatible dashboards w/ screenshots (supports png, jpeg, and gif images; relative paths only; e.g. `![](img/dashboard-1.png)` )-->
+
+<!-- This integration is compatible with the [{{dashboard_name}}][{{dashboard_link}}] (included w/ [Sensu Plus][sensu-plus]). -->
+
+<!-- ![](img/dashboard.png) -->
+
+There are no compatible dashboards for this integration.
+
+## Setup
+
+<!-- Sensu Integration setup instructions, including Sensu agent configuration and external component configuration -->
+<!-- EXAMPLE: what configuration (if any) is required in a third-party service to enable monitoring? -->
+
+1. This integration requires access to GCP Monitoring API.
+
+   All forms of GCP [authentication supported] by the [GCP Monitoring API] are supported, including [service account credentials]. This integration requires read-only access to the [GCP Monitoring API].
+
+   `GCP_MONITORING_CHECK_CREDS_JSON` secret or environment variable are unneeded if the check(s) from this integration are run on a `sensu-agent` installed on an GCP compute instance using a service account with an IAM Profile with necessary permissions to read the monitoring API.
```

Comment:
Consider simplifying this. The term "unneeded" is a bit of a stumbling block as well as the run-on nature of the sentence. Break this up into (at least) 2 sentences.

Reply:
updated in new commit